SpatialGDK/Source/SpatialGDK/Private/Tests/CrossServerRPCHandlerTest.cpp

```diff
@@ -22,39 +22,6 @@ const Worker_RequestId QueueingRequestId = 2;
 const FComponentSetData ComponentSetData = {};
 } // anonymous namespace
 
-class MockConnectionHandler : public AbstractConnectionHandler
```

Comment:
Sami delivered FTestWorker and other utilities for writing test. Maybe it could be worth using them, instead of moving this into the SpatialView headers, and spreading its usage ?